frontend/docs/docs/develop/docs.md

```diff
@@ -2,35 +2,52 @@
 
 Our documentation is built with [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) and configured via `mkdocs.yml` in the project root.
 
-The docs can be found in the `./docs` subdirectory.
+The docs can be found in the `frontend/docs` subdirectory.
 
-To build the docs locally, install Material for MkDocs with pip, pipx, or uvx:
+## Installation
```

Comment:
I wonder if we should even cover this here at all, since its also in the docs themselves: https://docs.browsertrix.com/develop/docs/

Reply:
It seems like a lot of space dedicated to deploying the docs

Reply:
Ack, I got confused I thought this was the README!